### DISCUSSION TITLE
A small question

### DISCUSSION THREAD

**Discussion by wuhao150313** [Halen84/ScriptHookRDR2DotNet-V2#18] [Q&A]
I am a computer newbie and obtained the address of V2 from version v1. However, when I unzipped v2, I found that there was no folder named "script" inside. 
  I'm sorry to disturb your time!!Really sorry!!
  But If I want to add mods, should I just drag them in or create a new folder named "script"? Or replace the V2 file with v1?Looking forward to receiving a reply.
<img width="1280" alt="dc38d697bb62ef75cd08525aac3609b" src="https://github.com/user-attachments/assets/6e47ee36-22a3-46a7-bb94-b5c121c227c2">
<img width="1280" alt="ca824f44b2cabc87689c85391171405" src="https://github.com/user-attachments/assets/f0022a2c-16c4-4de8-8632-2bd190530134">

**Comment by asciidude** [comment]
Create a folder named `scripts` and drag your .NET mods into there.
